stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/HttpUtil.java

```diff
@@ -211,22 +211,22 @@ public static void setEntity(Context context, boolean isRequest) {
     /**
      * Get the entity from request or response.
      *
-     * @param context                Ballerina context
-     * @param isRequest              boolean representing whether the message is a request or a response
-     * @param isEntityBodyRequired   boolean representing whether the entity body is required
+     * @param context            Ballerina context
+     * @param isRequest          boolean representing whether the message is a request or a response
+     * @param entityBodyRequired boolean representing whether the entity body is required
      * @return Entity of the request or response
      */
-    public static BValue[] getEntity(Context context, boolean isRequest, boolean isEntityBodyRequired) {
+    public static BValue[] getEntity(Context context, boolean isRequest, boolean entityBodyRequired) {
```

Comment:
Shall we maintain a single convention in naming boolean variables? Either with "is" prefix or not. Only the entityBodyRequired has changed, but not the isRequest

Reply:
Calling 'isRequest', "request" is misleading. I think disregarding the prefix is fine if the variable maintains a state in an object. Then we should use the proper java bean convention. But when we use it as a para, I don't think that getting rid of "is" prefix will improve the API. Please see section 14 in [1].

[1] https://www.cs.bgu.ac.il/~majeek/presentations/JavaProgrammingStyle%20Guidelines.html